src/Microsoft.Azure.CosmosRepository/Specification/Evaluator/OrderEvaluator.cs

```diff
@@ -18,7 +18,7 @@ internal class OrderEvaluator : IEvaluator
             where TItem : IItem
             where TResult : IQueryResult<TItem>
         {
-            if (specification.OrderExpressions == null)
+            if (specification.OrderExpressions.Any() is false)
```

Comment:
```suggestion
            if (specification?.OrderExpressions.Any() ?? false)
```

Is this check right? `Any is false` means it's empty, right?